tox.ini

```diff
@@ -6,7 +6,7 @@ envlist =
 [testenv]
 usedevelop = True
 commands =
-  core: python setup.py test {env:P_ARGS:}
+  core: python setup.py && python -m pytest test 
```

Comment:
According to:
https://docs.pytest.org/en/7.1.x/how-to/usage.html#calling-pytest-through-python-m-pytest

the command should be:
`python -m pytest test` 
without `python setup.py &&` part